### PR TITLE
Hacktober

### PR DESCRIPTION
**Pull request by Saga-sanga**
# Zero To Mastery - Webblocks 2022 📦📦

## PR contains:
- Fix to thank you card overlaying incorrectly over the rating card in mobile view.

## Breaking changes
- None

## Screenshots
- None

### CODE REVIEW

File: src/blocks/saga-sanga/index.css [hunk 1]
```diff
@@ -194,11 +194,11 @@ label:focus {
...
 .thank-you-section {
   display: none;
-  position: absolute;
+  /* position: absolute; */
   justify-items: center; 
   text-align: center;
-  align-self: center;
-  justify-self: center;
+  /* align-self: center; */
+  /* justify-self: center; */
```
Comment (sophiabrandt) [inline src/blocks/saga-sanga/index.css:201]: Same in line 200 & 201.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: src/blocks/saga-sanga/index.css [hunk 2]
```diff
@@ -194,11 +194,11 @@ label:focus {
...
   z-index: 2;
   /* padding-block: 2rem; */
```
[the remark this answers is not in the document]
Comment (Saga-sanga) [reply src/blocks/saga-sanga/index.css:203]: All right. Will do.

**Comment by Saga-sanga** [comment]
@sophiabrandt I have made the requested changes.

**Review by MattCSmith — APPROVED** [review]
Looks good to me 👍

**Pull request merged**